pkg/scheduler/framework/interface.go

```diff
@@ -160,6 +162,33 @@ func (s *Status) AsError() error {
 	return errors.New(s.Message())
 }
 
+// IsStatusEqual returns whether two Statuses are equal to each other.
+func IsStatusEqual(statusX, statusY *Status) bool {
```

Comment:
`x, y *Status` should be good enough.

Reply:
I do not have strong opinions about this. Just follow some existing pattern in our code. But since we will expose this func, I will follow your suggestions to make it more clear.

https://github.com/kubernetes/kubernetes/blob/b3dd01da497f4f9f81ffb2ebf275ff08cbfebaa2/pkg/scheduler/framework/runtime/registry_test.go#L74-L97